src/services/cosmos.ts

```diff
@@ -16,23 +16,16 @@ interface ValidatorReward {
 }
 
 export const cosmosService = {
-  getKiiAddress: async (evmAddress: string) => {
-    const response = await fetch(
-      `${API_ENDPOINTS.LCD}/kiichain/evm/kii_address?evm_address=${evmAddress}`
-    );
-    return await response.json();
-  },
-
   getDelegations: async (kiiAddress: string) => {
     const response = await fetch(
-      `${API_ENDPOINTS.LCD}/cosmos/staking/v1beta1/delegations/${kiiAddress}`
+      `${CHAIN_LCD_ENDPOINT}/cosmos/staking/v1beta1/delegations/${kiiAddress}`
     );
     const data = await response.json();
     return (
       data.delegation_responses?.reduce(
         (acc: number, curr: DelegationResponse) => {
           const amount = curr.balance?.amount
-            ? parseFloat(curr.balance.amount) / 1_000_000
+            ? parseFloat(curr.balance.amount) / 1_000_000_000_000_000_000
```

Comment:
As the other comment, turn the number to convert into a constant and this conversion function into a util function